spec/integration/inventory_spec.rb

```diff
@@ -152,7 +152,7 @@ module Hawkular::Inventory::RSpec
         it 'Should list resources for feed' do
           resources = @client.list_resources_for_feed feed_id
 
-          expect(resources.size).to be(2)
+          expect(resources.size).to be >= 2
```

Comment:
Just take in considerations that the same tests also run on every `hawkular-services` PR  ( @pilhuhn asked me to do that  ) as a way to early check for changes that would break compatibility with the client.

As of inventory.3 we are not longer able to write data into inventory (yet), the only capable of that are the agents.

We can make these tests predictable by updating hawkular-agent-configuration.yaml